apiserver/facades/agent/uniter/service.go

```diff
@@ -56,11 +56,25 @@ type CredentialService interface {
 
 // ApplicationService provides access to the application service.
 type ApplicationService interface {
+	// GetApplicationLife looks up the life of the specified application
```

Comment:
Be consistent with these comments. You included a period at the end of one, but not the others.

Reply:
That's because these comments were copied from somewhere else. I'll fix there as well!